### PR TITLE
creating-adaptive

### PR DESCRIPTION
**Pull request by Konmly**
Creating an adaptive for size 1440.

localhost_3000_

### CODE REVIEW

File: src/scss/_diver.scss [hunk 1]
```diff
@@ -8,9 +8,21 @@
 
   padding-top: 460px;
 
+  @media screen and (max-width: $xxl) {
+    padding-top: 350px;
+    background: url('../images/diver_bubble.png') 100% no-repeat,
```
Comment (PahaTonk) [inline src/scss/_diver.scss:13]: try to change only `background-image`

File: src/scss/_octopus.scss [hunk 2]
```diff
@@ -7,11 +7,19 @@
   background-repeat: no-repeat, no-repeat, no-repeat;
   background-position: left 128px bottom 109px, right 27px bottom 31px, top 0 left 0;
 
+  @media screen and (max-width: $xxl) {
+    padding-bottom: 83px;
```
Comment (PahaTonk) [inline src/scss/_octopus.scss:11]:
```suggestion
```
`1px` we can leave out

File: src/scss/_styles.scss [hunk 3]
```diff
@@ -24,6 +24,15 @@ body {
...
   margin: 0 auto;
+
+  @media screen and (max-width: $xxl) {
+    width: 1120px ;
+    margin: 0 auto;
+  }
+
+  @media screen and (max-width: $xl) { //* это я считала размер ширины на 1024 по пропорции, пока это не используется*\\
+    width: 820px;
+  }
```
Comment (PahaTonk) [inline src/scss/_styles.scss:35]:
```suggestion
/* TODO-[YOUR_SURNAME]: [YOUR_COMMENT_ON_ENGLISH] */
/*  @media screen and (max-width: $xl) {  */
/*    width: 820px; */
/*  } */
```
What is not being used is wrapped in a comment

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: src/scss/_diver.scss (hunk 1 above)
Comment (Konmly) [reply src/scss/_diver.scss:13]: then there will be no gradient on the background
q1

**Review by PahaTonk — APPROVED** [review]

**Pull request merged**